src/blocks/cpu.rs

```diff
@@ -46,12 +49,16 @@ pub struct CpuConfig {
     #[serde(default = "CpuConfig::default_critical")]
     pub critical: u64,
 
-    /// Display frequency
-    #[serde(default = "CpuConfig::default_frequency")]
-    pub frequency: bool,
+    /// Format override
+    #[serde(default = "CpuConfig::default_format")]
+    pub format: String,
 }
 
 impl CpuConfig {
+    fn default_format() -> String {
+        "{barchart} {utilization}% {frequency}GHz".to_owned()
```

Comment:
We should probably match the existing block output as the default, which I believe is equivalent to `{utilization}%`.